FanTheorem.agda

```diff
@@ -55,8 +53,8 @@ module _ (𝔅 : ℘ (Neigh 𝟚)) (𝔅? : ∀ U → Decidable (𝔅 U)) (⊨
 
   𝔅⊑𝔄 : 𝔅 ⊑ 𝔄
   𝔅⊑𝔄 𝔅[U] =
-    1 , λ α →
-      ze ,
+    1 ▸ λ α →
```

Comment:
thanks for changing this operator! Was kind of frustrating to have to disambiguate it from the other one